packages/app/obojobo-document-engine/src/scripts/viewer/components/viewer-app.js

```diff
@@ -255,6 +257,7 @@ export default class ViewerApp extends React.Component {
 
 		// use Focus Store values to update DOM Focus
 		this.updateDOMFocus()
+		this.startObservingForIntersectionChanges()
```

Comment:
Need a parallel `stopObservering...`, this probably isn't too bad but I'm concerned that we're not disconnecting the observers. I don't know the reality but it seems possible to create memory leaks.